Sdk/Tests/ApiTest.php

```diff
@@ -53,8 +57,10 @@ public function testConstructorWithoutOption()
 
     public function testGetProjects()
     {
-        $this->pluginMockResponse->addResponse($this->createResponse('projects'));
-        $projects = $this->api->getProjects();
+        $client = new MockHttpClient([new MockResponse($this->createResponse('projects'))]);
+        $api = new Api(array('api_token' => 'my-token', 'user_uuid' => 'my-user-uuid'), $client, null, $this->logger);
```

Comment:
You could wrap these two lines in a method like `private function createApi(array $responses)`, then use it here (`$api = $this->createApi([new MockResponse($this->createResponse('projects'))])`)